randovania/games/dread/logic_database/Artaria.txt

```diff
@@ -4829,18 +4829,36 @@ Extra - asset_id: collision_camera_081
   * Extra - left_shield_def: actordef:actors/props/doorshieldmissile/charclasses/doorshieldmissile.bmsad
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
-  > Next to Upper Tank
+  > Start Point 2
       All of the following:
-          Before Artaria - Screw Attack Rotatable and Enabled Highly Dangerous Logic
+          Speed Booster and Speed Booster Conservation (Beginner) and Disabled Door Lock Randomizer and Shoot Missile
           Any of the following:
-              Flash Shift or Movement (Beginner) or Lay Cross Bomb
-              Speed Booster and Shoot Missile
+              # https://youtu.be/VgTURzSZz8I
+              Screw Attack
+              # https://youtu.be/T4hIL-R_0p0
+              Morph Ball and Slide and Movement (Beginner)
+  > Next to Upper Tank
+      Any of the following:
+          All of the following:
+              Before Artaria - Screw Attack Rotatable and Enabled Highly Dangerous Logic
+              Any of the following:
+                  Flash Shift or Movement (Beginner) or Lay Cross Bomb
+                  Speed Booster and Disabled Door Lock Randomizer and Shoot Missile
+          All of the following:
+              Speed Booster and After Artaria - Screw Attack Rotatable and Speed Booster Conservation (Beginner) and Disabled Door Lock Randomizer and Shoot Missile
+              Any of the following:
+                  # https://youtu.be/F2L7qs1uP90
+                  Screw Attack
+                  # https://youtu.be/QJuxx0SSQMI
+                  Gravity Suit and Morph Ball
   > Event - Screw Attack Rotatable
       Trivial
   > Under left side blocks
       All of the following:
           After Artaria - Screw Attack Rotatable
           Morph Ball or Screw Attack
+  > Event - Screw Attack Rotatable to left of water
+      Speed Booster and Before Artaria - Screw Attack Rotatable and Speed Booster Conservation (Beginner) and Disabled Door Lock Randomizer and Enabled Highly Dangerous Logic and Shoot Missile
```

Comment:
With Screw Attack, this would not actually be dangerous. So it could be either Highly Dangerous Logic or Screw Attack.

Reply:
After careful consideration, I added Screw Attack here, and I ended up restricting the hand waving on flipping the spinner to when highly dangerous logic is disabled. I think this should cover most of the bases, as I think it's still effectively in logic to get the top item with screw attack and some tricks outside of highly dangerous logic this way.